tests/handlers/protocols-configure.spec.ts

```diff
@@ -289,6 +292,147 @@ export function testProtocolsConfigureHandler(): void {
         expect(protocolsConfigureReply.status.detail).to.contain(DwnErrorCode.AuthorizationAuthorNotOwner);
       });
 
+      it('should reject ProtocolsConfigure with action rule containing duplicated actor (`who`) within a rule set', async () => {
```

Comment:
Suggestion:
```suggestion
      it('should reject ProtocolsConfigure with action rule containing duplicated actor (`who`) or (`of`) within a rule set', async () => {
```

Or potentially split it up into two tests.

Reply:
Done. Reworded for better clarity.